doc/plugins/authoring.rst

```diff
@@ -113,6 +113,46 @@ Important Notes
              os.path.dirname(os.path.abspath(__file__)) + "/nix"
          ]
 
+5. Resource subclasses must now work with Python objects instead of XML
+
+   This old-style ResourceDefinition subclass:
+
+   .. code-block:: python
+
+     class MachineDefinition(nixops.resources.ResourceDefinition):
```

Comment:
We might want to make this more specific. Like, `NeatCloudMachineOptions`, to hint that it is specific to their module. (I imagine instructions like this will get a lot of copy-paste.)